File: app/src/main/java/com/sd/a3kleingroup/MainActivity.java
package com.sd.a3kleingroup;

import android.content.Intent;
import android.os.Bundle;
import android.os.PersistableBundle;
import android.util.Log;
import android.widget.Toast;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState, @Nullable PersistableBundle persistentState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toast.makeText(this.getApplicationContext(), "Toast", Toast.LENGTH_LONG).show();
        Log.d("LOG_MAIN_ACT", " In here");
        // test login
        Intent intent2 = new Intent(this, LoginActivity.class);
        startActivity(intent2);

    }
}

File: app/src/main/java/com/sd/a3kleingroup/MainActivity.kt
package com.sd.a3kleingroup

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main);

        // test login
//        val intent2 = Intent(this, LoginActivity::class.java)
//        startActivity(intent2);
//        return;
        // just go to the sendFile act
        // todo remove
//        val intent = Intent(this, SendFileActivity::class.java)
//        startActivity(intent);

        val intent = Intent(this, ReceiveFilesActivity::class.java)
        startActivity(intent);
    }
    public fun sayGoodbye(): String{
        return "Goodbye"
    }
}
